build/Dockerfile

```diff
@@ -1,47 +1,43 @@
-FROM ubuntu:15.04
-LABEL version=0.0.6
+FROM centos:6
+LABEL version=0.1.1
 
-RUN sed -i -e 's/archive.ubuntu.com\|security.ubuntu.com/old-releases.ubuntu.com/g' -e 's/us\.old/old/g' /etc/apt/sources.list && apt-get clean
+# Install dependencies for developer tools, bindings,\
+# documentation, actorcompiler, and packaging tools\
+RUN yum install -y yum-utils &&\
+	yum-config-manager --enable rhel-server-rhscl-7-rpms &&\
+	yum -y install centos-release-scl epel-release &&\
+	yum -y install devtoolset-7 mono-core java-1.8.0-openjdk-devel \
+		rh-python36-python-devel rh-ruby24 golang python27 \
+		rpm-build debbuild python-pip npm ccache distcc
 
-RUN apt-get update && apt-get --no-install-recommends install -y --force-yes bzip2 ca-certificates=20141019 adduser apt base-files base-passwd bash binutils build-essential cpp cpp-4.9 dpkg dos2unix elfutils fakeroot findutils g++=4:4.9.2-2ubuntu2 g++-4.9=4.9.2-10ubuntu13 gawk=1:4.1.1+dfsg-1 gcc-5-base gcc=4:4.9.2-2ubuntu2 gcc-4.9=4.9.2-10ubuntu13 gcc-4.9-base:amd64=4.9.2-10ubuntu13 gcc-5-base:amd64=5.1~rc1-0ubuntu1 gdb git golang golang-go golang-go-linux-amd64 golang-src grep gzip hostname java-common libasan1 liblsan0 libtsan0 libubsan0 libcilkrts5 libgcc-4.9-dev libstdc++-4.9-dev libgl1-mesa-dri libgl1-mesa-glx libmono-system-xml-linq4.0-cil libmono-system-data-datasetextensions4.0-cil libstdc++-4.9-pic locales login m4 make makedev mawk mono-dmcs npm openjdk-8-jdk passwd python-distlib python-gevent python-greenlet python-html5lib python-minimal python-pip python-pkg-resources python-requests python-setuptools python-six python-urllib3 python-yaml python2.7 python2.7-minimal rpm rpm2cpio ruby ruby2.1 rubygems-integration sed tar texinfo tzdata-java udev unzip util-linux valgrind vim wget golang-go.tools curl sphinx-common gnupg python-dev python3 python3-dev
-
-RUN adduser --disabled-password --gecos '' fdb && chown -R fdb /opt && chmod -R 0777 /opt
+RUN adduser --comment '' fdb && chown -R fdb /opt && chmod -R 0777 /opt
 
 USER fdb
 
 # wget of bintray without forcing UTF-8 encoding results in 403 Forbidden
 RUN cd /opt/ &&\
```

Comment:
I would generally prefer to put dependencies into `/usr/local`. Otherwise we'll need to modify the `PATH` variable to make this work.

Reply:
We have a need to be able to install multiple versions such that the boost root directory: ie boost_1_67 would need to be included. For that reason, we use /opt rather than /usr/local (though that could also be done)

Reply:
That makes sense to me - and I guess having to pass `-DBOOST_ROOT` in the docker file is not a big deal